<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleWidget</class>
 <widget class="QWidget" name="RuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RuleWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ruleBeep">
       <property name="accessibleDescription">
        <string>TOGGLE_SOUND</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="QtResource.qrc">
         <normaloff>:/Resources/Mute.png</normaloff>:/Resources/Mute.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="QtResource.qrc">
         <normaloff>:/Resources/ArrowUp.png</normaloff>:/Resources/ArrowUp.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="QtResource.qrc">
         <normaloff>:/Resources/ArrowDown.png</normaloff>:/Resources/ArrowDown.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelGroupID">
       <property name="accessibleName">
        <string>GROUP_ID</string>
       </property>
       <property name="text">
        <string>Group ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="groupID">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleSave">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="QtResource.qrc">
         <normaloff>:/Resources/Save.png</normaloff>:/Resources/Save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="ruleSequencialMode">
       <property name="accessibleName">
        <string>SEQUENTIAL_MODE</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton:disabled:checked {color: black;}</string>
       </property>
       <property name="text">
        <string>Sequential Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ruleConcurrentMode">
       <property name="accessibleName">
        <string>CONCURRENT_MODE</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton:disabled:checked {color: black;}</string>
       </property>
       <property name="text">
        <string>Concurrent Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="rulesTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>26</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="rulesNoMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="accessibleName">
      <string>RULE_NOITEMS</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray; background: white; color: gray</string>
     </property>
     <property name="text">
      <string>No Rules</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ruleAddButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>TR00027</string>
       </property>
       <property name="text">
        <string>Add Rule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleEditButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>RULE_EDIT</string>
       </property>
       <property name="text">
        <string>Edit Rule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ruleEnableDisable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>ENABLE_DISABLE</string>
       </property>
       <property name="text">
        <string>Enable / Disable</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_32">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>REMOVE_RULES</string>
       </property>
       <property name="text">
        <string>Remove:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleRemoveAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>CANCEL_ALL_ORDERS</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ruleRemove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="accessibleName">
        <string>CANCEL_SELECTED_ORDERS</string>
       </property>
       <property name="text">
        <string>Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rulesTable</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>ruleEditButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
